### DISCUSSION TITLE
Getting the home of an element via ref for vue 3

### DISCUSSION THREAD

**Discussion by ghost** [vuejs/docs#1641] [Q&A]
Faced with the following problem:

When iterating through and rendering data via v-for, it is not possible to access the home element via ref.

Vue js version - 3.2.31

The documentation states that it can be obtained as follows - https://vuejs.org/guide/essentials/template-refs.html#refs-inside-v-for

Example from the documentation:

![example2](https://user-images.githubusercontent.com/58338625/160226224-bd0191b8-2af0-48ce-b077-c73788aad579.jpg)

My code example:

![example1](https://user-images.githubusercontent.com/58338625/160226255-f2060a63-c82d-4c9b-8b24-6b830a4447d3.jpg)

![example3](https://user-images.githubusercontent.com/58338625/160226282-87e131e3-60be-4106-ac07-ef0cc7afd18f.jpg)

![example4](https://user-images.githubusercontent.com/58338625/160226285-75dad246-6ce5-44e3-a96b-12e819ec88ab.jpg)

but for some reason, it doesn't work.

Maybe someone has already faced such a problem?

**Comment by farid-temuri-lofty** [comment]
works for me [vue sfc playground](https://sfc.vuejs.org/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCB7b25Nb3VudGVkLCByZWZ9IGZyb20gJ3Z1ZSdcbiAgXG5jb25zdCBpdGVtUmVmID0gcmVmKFtdKVxuY29uc3QgbGlzdCA9IHJlZihbMSwyLDNdKVxuIFxub25Nb3VudGVkKCgpPT4ge1xuICBjb25zb2xlLmxvZyhpdGVtUmVmKVxufSlcbiAgXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8dWw+XG4gICAgPGxpIHYtZm9yPVwiaXRlbSBpbiBsaXN0XCIgcmVmPVwiaXRlbVJlZlwiPlxuICAgICAge3sgaXRlbSB9fVxuICAgIDwvbGk+XG4gIDwvdWw+XG48L3RlbXBsYXRlPiIsImltcG9ydC1tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHtcbiAgICBcInZ1ZVwiOiBcImh0dHBzOi8vc2ZjLnZ1ZWpzLm9yZy92dWUucnVudGltZS5lc20tYnJvd3Nlci5qc1wiLFxuICAgIFwidnVlL3NlcnZlci1yZW5kZXJlclwiOiBcImh0dHBzOi8vc2ZjLnZ1ZWpzLm9yZy9zZXJ2ZXItcmVuZGVyZXIuZXNtLWJyb3dzZXIuanNcIlxuICB9XG59In0=)
